docs/src/main/sphinx/appendix/from-hive.rst

```diff
@@ -110,6 +110,23 @@ Trino query::
     FROM tests
     CROSS JOIN UNNEST(scores) AS t (score);
 
+Use ANSI SQL syntax for date and time INTERVAL expressions
+----------------------------------------------------------
+
+Trino supports the ANSI SQL style INTERVAL expressions and this differs from the implementation used in Hive:
+
+* The INTERVAL keyword is required and is not optional.
+* Date and time units must be singular e.g. day and not days.
+* Values should be quoted and not bare strings e.g. '14' and not 14.
```

Comment:
Values must be quoted.

The rest is clear from the following example.

Reply:
Fixed